designer/server/src/test/scala/pl/touk/nussknacker/ui/api/ManagementResourcesSpec.scala

```diff
@@ -242,7 +242,7 @@ class ManagementResourcesSpec extends AnyFunSuite with ScalatestRouteTest with F
         status shouldBe StatusCodes.Conflict
       }
       getProcess(ProcessName(invalidScenario.id)) ~> check {
-        decodeDetails.state.value.status shouldEqual SimpleStateStatus.NotDeployed
+        decodeDetails.state.value.status.name shouldEqual SimpleStateStatus.NotDeployed.name
```

Comment:
Why you don't compare it to whole case object?

Reply:
decodeDetails.state.value.status is created ad-hoc and is different from SimpleStateStatus.NotDeployed